### ISSUE TITLE
Can not Clear Old Bullet when using LineSeries with Bullet and Enabling grouping

### ISSUE THREAD

**Issue by saleemshenlin** [amcharts/amcharts4#1908]
if you use lineseries with bullet, and enable group, chart can not clean old data bullets when changing new chart data.
``` javascript
![aRjy2mF9rT](https://user-images.githubusercontent.com/3690320/69404249-ca807500-0d37-11ea-925b-06b44cf42ed1.gif)

/**
 * ---------------------------------------
 * This demo was created using amCharts 4.
 * 
 * For more information visit:
 * https://www.amcharts.com/
 * 
 * Documentation is available at:
 * https://www.amcharts.com/docs/v4/
 * ---------------------------------------
 */

// Themes begin
am4core.useTheme(am4themes_animated);
// Themes end

var chart = am4core.create("chartdiv", am4charts.XYChart);
chart.paddingRight = 40;

function makeData(){
  var data = [];
var value = 50;
for (let i = -3650; i < 0; i++) {
  let date = new Date();
  date.setHours(0, 0, 0, 0);
  date.setDate(i);
  value -= Math.round((Math.random() < 0.5 ? 1 : -1) * Math.random() * 10);
  if (value < 0) {
    value = Math.round(Math.random() * 10);
  }
  data.push({ date: date, value: value });
}
  return data;
}

setInterval(function(){
   chart.data = makeData();
}, 1500);

// Create axes
var dateAxis = chart.xAxes.push(new am4charts.DateAxis());
dateAxis.renderer.minGridDistance = 60;
dateAxis.groupData = true;

var valueAxis = chart.yAxes.push(new am4charts.ValueAxis());

// Create series
var series = chart.series.push(new am4charts.LineSeries());
series.dataFields.valueY = "value";
series.dataFields.dateX = "date";
series.tooltipText = "{value}"
series.strokeWidth = 2;
const bullet = series.bullets.push(new am4charts.CircleBullet());
bullet.circle.radius = 2;
bullet.fill = measureSeries.fill;
bullet.circle.strokeWidth = 0;

series.tooltip.pointerOrientation = "vertical";

chart.cursor = new am4charts.XYCursor();
chart.cursor.snapToSeries = series;
chart.cursor.xAxis = dateAxis;

chart.scrollbarX = new am4core.Scrollbar();
```
![aRjy2mF9rT](https://user-images.githubusercontent.com/3690320/69404273-dbc98180-0d37-11ea-855d-ba5866209a07.gif)

**Comment by martynasma** [comment]
Looks like aggregate data is not being cleared across data updates. Thanks for the heads up!

Example for the dev team:
https://codepen.io/team/amcharts/pen/d2dc5b53c7b34b27456c3a6d4de10cad?editors=0010

**Comment by saleemshenlin** [comment]
i have tested many ways, and using 'series.clear' is the only solution.
```  typescript
chart.series.clear();
```

**Comment by develar** [comment]
Workaround that doesn't affect performance:

```
for (const series of chart.series) {
  series.bulletsContainer.disposeChildren()
}
```

**Comment by depyronick** [comment]
this problem still exist

**Comment by Stepik42** [comment]
Any updates here? Data grouping with bullets still not working.

**Comment by martynasma** [comment]
Fixed in [4.9.35](https://github.com/amcharts/amcharts4/releases/tag/4.9.35).

## [4.9.35] - 2020-08-12

### Added
- New adapter on `XYSeries`: `groupValue`. It's now possible to apply custom aggregate value calculation functions for grouped data items. [More info](https://www.amcharts.com/docs/v4/tutorials/using-custom-functions-for-data-item-grouping/).
- New adapter on `Cursor`: `cursorPoint`. Use it to custom-position chart cursor using your own logic.

### Changed
- Accessibility: Now if ENTER is pressed when object with an enabled `"hit"` event is focused, that event will be triggered automatically.
- Clicking anywhere on the plot area with `XYCursor` will now trigger blur on any focused element on the page (consistent to clicking anywhere else on the document).

### Fixed
- RangeSelector plugin was not properly inheriting chart's locale.
- `"x"` (timestamp) in `inputDateFormat` was not working properly.
- If all values of `SankeyDiagram` were `0` (zero), nodes were not being arranged and their rectangles were not being displayed.
- First `CategoryAxis` label was not visible if `minGridDistance` was set to `0` (zero).
- Old series bullets of an `XYSeries` remained visible when data was updated and data grouping was enabled on related `DateAxis`.
- `Utils.random()` function was not returning value in proper from/to range.
- Stacking now works only between series of the same type. Previously a `LineSeries` between two stacked `ColumnSeries` could mess up stacking for the latter.
- Zooming in on a part of `LineSeries` which had `strokeDashArray` enabled via `propertyFields` was making the whole line dashed.
- On a chart with `XYCursor` hovering on category with a null value for some series, used to display a tooltip for a different category in some cases.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
